configure.ac

```diff
@@ -482,6 +482,9 @@ fi
 # For hddtemp module
 AC_CHECK_HEADERS(linux/major.h)
 
+# For hugepages module
+AC_CHECK_HEADERS(dirent.h)
```

Comment:
Scratch that, we already include dirent.h in collectd.h and ./configure already looks for it, so you can safely assume it exists.